examples/basic_bitcoin/test/tests.ts

```diff
@@ -21,17 +21,19 @@ const FIRST_AMOUNT_SENT = SINGLE_BLOCK_REWARD / 2n;
 const SECOND_AMOUNT_SENT = SINGLE_BLOCK_REWARD * 2n;
 const TO_ADDRESS = 'n4HY51WrdxATGEPqYvoNkEsTteRfuRMxpD';
 
-let lastTxid = '';
-let toAddressPreviousBalance = 0n;
-let canisterPreviousBalance = 0n;
+type BasicBitcoinContext = {
```

Comment:
All of these should be optional...but actually null?

Reply:
Let's try to do the crazy context type changing thing, otherwise make them optional/undefined